.env.example-full

```diff
@@ -41,3 +41,14 @@ IMAGE_HOSTING_SPACE_ACCESS_KEY_ID=''
 IMAGE_HOSTING_SPACE_SECRET_ACCESS_KEY=''
 
 OPENAI_API_KEY=''
+
+# enable search
```

Comment:
might be worth to make it
```suggestion
# enable search (readonly)
```
to indicate clearly that these two are only used for querying (and are public information), never for writing to the index.